### DISCUSSION TITLE
Hide object list in the room for invited users.

### DISCUSSION THREAD

**Discussion by orcstar** [mozilla/hubs#4087] [General Questions & Discussion] [answered]
Hello!
I would like to hide / remove the object list of one room for invited people, is that posible? I'm trying to create a "game" where users will have to find some hidden objects, and the object list makes it to easy to find them.
Thanks a lot!

**Comment by johnshaughnessy** [comment, marked as answer]
There is currently no option to hide or disable the object list. You would need to change the code in a custom client. See https://hubs.mozilla.com/docs/hubs-cloud-custom-clients.html for details
